### ISSUE TITLE
Insert Image into CodeComment not possible via ClipBoard

### ISSUE THREAD

**Issue by 6543** [go-gitea/gitea#27960]
**Gitea versions**: v1.18.0 ... v1.20.0

You can insert images from the clipboard using `ctrl+v` into issue/pull/comment content, or drag and drop them. However, this does not work with code comments.

Of course, you could upload it as an attachment to a normal comment and then use it in the code comment via normal markdown syntax, but this hinders user productivity.

**Comment by JakobDev** [comment]
So you mean you drag and drop a code file in the editor and the editor creates a Codeblock?
